src/utils/helpers/style-helper/index.js

```diff
@@ -1,4 +1,4 @@
-import { isClassic } from './style-helper';
+import { isClassic, isDLS } from './style-helper';
 
 // eslint-disable-next-line import/prefer-default-export
```

Comment:
Please could you remove the linter disable commands now that we don't need them?